src/content/11/en/part11d.md

```diff
@@ -168,12 +173,20 @@ Complete the workflow and try it out! Once the workflow has run successfully, yo
 
 GitHub allows you to set up protected branches. It is important to protect your most important branch that should never be broken: master. In repository settings you can choose between several levels of protection. We will not go over all of the protection options, you can learn more about them in GitHub documentation. Requiring pull request approval when merging into master is one of the options we mentioned earlier.
 
-From CI point of view the most important protection is requiring status checks to pass before a PR can be merged into master. This means that if you have set up GitHub actions to run e.g. linting and testing tasks, then until all the lint errors are fixed and all the tests pass the PR cannot be merged.
+From CI point of view the most important protection is requiring status checks to pass before a PR can be merged into master. This means that if you have set up GitHub actions to run e.g. linting and testing tasks, then until all the lint errors are fixed and all the tests pass the PR cannot be merged. Because you are the administrator for you repository, you will see an option to override the restriction. However, non-administrators will not have this option.
 
-// TODO: screenshot of an unmergeable PR
+![Unmergeable PR](../../images/11/part11d_03.png)
 
 To set up protection for your master branch, navigate to repository "Settings" from the top menu inside the repository. In the left-side menu select "Branches". Click "Add rule" button next to "Branch protection rules". Type a branch name pattern ("master" will do nicely) and select the protection you would want to set up. At least "Require status checks to pass before merging" is necessary for you to fully utilise the power of GitHub Actions. Under it you should also check "Require branches to be up to date before merging" and select all of the status checks that should pass before a PR can be merged. At least bulding, testing and linting checks are good candidates.
 
-// TODO: screenshot of the branch protection UI
+![Branch protection rule](../../images/11/part11d_04.png)
+
+### Exercise 11.6.
```

Comment:
Exercise blocks should start with `<div class="tasks">` to get the look-n-feel that matches rest of the site.